common-libs/common/package-lock.json

```diff
@@ -10,6 +10,12 @@
       "license": "ISC",
       "dependencies": {
         "@affinidi/affinity-metrics-lib": "^0.0.25",
+        "@affinidi/internal-api-clients": "^2.0.0",
```

Comment:
do not put in package-lock.json dependencies from SDK monorepo - only to package.json

Reply:
to omit that use `npm run bootstrap` before commiting